webapp/packages/core-administration/src/AdministrationScreen/AdministrationScreenService.ts

```diff
@@ -230,6 +230,10 @@ export class AdministrationScreenService {
       return true;
     }
 
+    if (fromState.params.item === 'license' && toState.params.item === 'users') {
```

Comment:
I am not sure the CE product should know about the license route.

Reply:
yep, we can't use plugins related code in core-* packages, we need to specify special behaviour via `IAdministrationItemOptions` in `administrationItemService.create`